### PR TITLE
DOCS-3591: Add examples screenshot to payment methods

### PR DESCRIPTION
**Pull request by FrancescaNCook**
Do the changes meet MultiSafepay Docs' Definition of Done?
- Style guide, tone of voice, and naming conventions
- Tested
- Reviewed on mobile, tablet and desktop screens
- Notified stakeholders
- Pass GitHub Actions

### CODE REVIEW

File: content/payment-management/second-chance.md [hunk 1]
```diff
@@ -178,7 +178,7 @@ Most payment methods are supported, **except for**:
 
 Second Chance emails can create conflicts with external warehouse systems. In some cases, this can be resolved using a cron job. However, this is not always a stable solution.
 
-For example, when a customer cancels an order in the webshop, they can still pay for it using Second Chance within 30 days or a specified time frame. For more information, see API reference - [Create order](/reference/createorder/) > `days_active` parameter.
+For example, when a customer cancels an order in the webshop, they can still pay for it using Second Chance within 30 days or a specified time frame. For more information, see API reference - See API reference – [Create order](/reference/createorder/) > `days_active` parameter.
```
Comment (Sarah-MultiSafepay) [inline content/payment-management/second-chance.md:181]:
```suggestion
For example, when a customer cancels an order in the webshop, they can still pay for it using Second Chance within 30 days or a specified time frame. For more information, see API reference – [Create order](/reference/createorder/) > `days_active` parameter.
```

File: content/payment-methods/afterpay.md [hunk 2]
```diff
@@ -66,23 +66,41 @@ You need to contact AfterPay to activate it for your account.
...
-- [Shopware 5 and 6](/docs/shopware/)
-- [WooCommerce](/docs/woocommerce/)
-- [X-Cart](/docs/x-cart/)
+
+AfterPay is supported in many of our ready-made integrations.
+
+  <details id="example-requests"> 
+  <summary>Example requests</summary>
+  <br>
+
```
Comment (Sarah-MultiSafepay) [inline content/payment-methods/afterpay.md:91]:
```suggestion
```

File: content/payment-methods/afterpay.md [hunk 3]
```diff
@@ -66,23 +66,41 @@ You need to contact AfterPay to activate it for your account.
...
+  - [Drupal 8](/docs/drupal/)
+  - [Magento 1](/docs/magento-1/) & [Magento 2](/docs/magento-2/)
+  - [Odoo](/docs/odoo/)
+  - [OpenCart](/docs/opencart/)
+  - [PrestaShop 1.6 and 1.7](/docs/prestashop-1-7/)
+  - [Shopware 5 and 6](/docs/shopware/)
+  - [WooCommerce](/docs/woocommerce/)
+  - [X-Cart](/docs/x-cart/)
+
+  </details>
```
Comment (Sarah-MultiSafepay) [inline content/payment-methods/afterpay.md:103]:
```suggestion
```

File: content/payment-methods/google-pay.md [hunk 4]
```diff
@@ -83,6 +83,17 @@ First apply to MultiSafepay, and then activate in your dashboard.
 
 ### API
 - To easily integrate Google Pay using MultiSafepay payment pages (redirect), see API reference – [Create order](/reference/createorder/) > Wallet order.
+
+  <details id="example-requests"> 
+  <summary>Example requests</summary>
+  <br>
+
+  For example requests, on the [Create order](/reference/createorder/) page, in the black sandbox, see **Examples** > **Apply Pay direct/redirect**.
```
Comment (Sarah-MultiSafepay) [inline content/payment-methods/google-pay.md:91]:
```suggestion
  For example requests, on the [Create order](/reference/createorder/) page, in the black sandbox, see **Examples** > **Google Pay direct/redirect**.
```

File: content/payment-methods/manual-capture.md [hunk 5]
```diff
@@ -27,7 +27,7 @@ See Recipes:
 - [Manual capture: Capture payment](/recipes/manual-capture-capture-payment/)
 
 See API reference:
-- [Create order](/reference/createorder/) > Set `manual` to `capture`.
+- See API reference – [Create order](/reference/createorder/) > Set `manual` to `capture`.
```
Comment (Sarah-MultiSafepay) [inline content/payment-methods/manual-capture.md:30]: Do we need to duplicate this as we already have "See API reference:" just before?

File: content/payment-methods/apple-pay-integration.md [hunk 6]
```diff
@@ -241,7 +241,7 @@ With redirect integration, customers are redirected to a [payment page](/docs/pa
 
 See:
 
-- [Create order](/reference/createorder/) > Wallet order
+- See API reference – [Create order](/reference/createorder/) > Wallet order
```
Comment (Sarah-MultiSafepay) [inline content/payment-methods/apple-pay-integration.md:244]:
```suggestion
- API reference – [Create order](/reference/createorder/) > Wallet order
```

File: content/payment-management/second-chance.md (hunk 1 above)
Comment (FrancescaNCook) [inline content/payment-management/second-chance.md:181]:
```suggestion
For example, when a customer cancels an order in the webshop, they can still pay for it using Second Chance within 30 days or a specified time frame. For more information, see API reference – [Create order](/reference/createorder/) > `days_active` parameter.
```

File: content/payment-methods/apple-pay-integration.md [hunk 7]
```diff
@@ -241,7 +241,7 @@ With redirect integration, customers are redirected to a [payment page](/docs/pa
 
 See:
```
Comment (FrancescaNCook) [inline content/payment-methods/apple-pay-integration.md:242]:
```suggestion
```

File: content/payment-methods/manual-capture.md (hunk 5 above)
Comment (FrancescaNCook) [inline content/payment-methods/manual-capture.md:30]:
```suggestion
- [Create order](/reference/createorder/) > Set `manual` to `capture`.
```

**Review by Sarah-MultiSafepay — APPROVED** [review]

**Pull request merged**